test/test-bugs.js

```diff
@@ -292,14 +292,14 @@
         
         $.mockjax({
             type: 'GET',
-            url: 'http://jquery-mockjax-foobar.com/somefile.js',
+            url: 'http://crossorigin.test/somefile.js',
```

Comment:
I saw the access control header in the grunt file was added for this domain, but why do we need to change the domain? That is, why not just add the access control header to match the old domain? I don't really care about, just wondering if there was a reason.

Reply:
It's not strictly required, no - and in fact I'm not certain yet whether CORS adjustments are required at all yet, but I figured it'd be worth using an IETF-reserved test top-level-domain name for safety here, now that those exist.